examples/api/lib/widgets/heroes/hero.1.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for Hero
+
+import 'package:flutter/material.dart';
+import 'package:flutter/scheduler.dart';
+
+void main() {
+  // Slow down time to see Hero flight animation.
+  timeDilation = 10.0;
+  runApp(const HeroApp());
+}
+
+class HeroApp extends StatelessWidget {
+  const HeroApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(title: const Text('Hero Sample')),
+        body: const Center(
+          child: HeroExample(),
+        ),
+      ),
+    );
+  }
+}
+
+class HeroExample extends StatelessWidget {
+  const HeroExample({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: <Widget>[
+        ListTile(
+          leading: Hero(
+            tag: 'hero-default-tween',
+            child: _blueRectangle(50.0),
+          ),
+          onTap: () => _gotoDetailsPage(context),
+          title:
+              const Text('Tap on the icon to view hero flight animation with default rect tween'),
+        ),
+        const SizedBox(
+          height: 20.0,
+        ),
+        ListTile(
+          leading: Hero(
+            tag: 'hero-custom-tween',
+            createRectTween: (Rect? begin, Rect? end) {
+              return MaterialRectCenterArcTween(begin: begin, end: end);
```

Comment:
In the video, the two options actually look very similar. Should we spice it up and make it more obvious that they are different?

Reply:
Thanks for taking a look!

I just updated the example with transparent color and overlapping alignment for easier comparison